datadog_checks_base/tests/test_http.py

```diff
@@ -423,6 +423,13 @@ def test_no_proxy_uris_coverage(self):
 
         http.get('https://www.google.com')
 
+    def test_socks5_proxy(self, socks5_proxy):
```

Comment:
Need to not run on appveyor, like https://github.com/DataDog/integrations-core/blob/0ff57202473f230c844e10dd24a6cb1910b9cd99/datadog_checks_dev/tests/test_docker.py#L13